Test/Sampler/test_sampler.py

```diff
@@ -26,7 +26,9 @@
 # Hilbert space of spins from given graph
 hi = nk.hilbert.Spin(s=0.5, N=g.n_nodes)
 ha = nk.operator.Ising(hilbert=hi, graph=g, h=1.0)
-move_op = sum([nk.operator.spin.sigmax(hi, i) for i in range(hi.size)])
+move_op = sum(
+    [nk.operator.LocalOperator(hi, [[0, 1], [1, 0]], i) for i in range(hi.size)]
+)
```

Comment:
please drop those lines

Reply:
Maybe I did not rebase my branch with the new master locally because I'm seeing some tests with old code in this PR... I will try to fix this